node/src/Commands.ts

```diff
@@ -3929,3 +3929,22 @@ export function createGetEx(
 
     return createCommand(RequestType.GetEx, args);
 }
+
+/**
+ * @internal
+ */
+export function createXGroupSetid(
+    key: string,
+    groupName: string,
+    id: string,
+    entriesRead?: number,
+): command_request.Command {
+    const args = [key, groupName, id];
+
+    if (entriesRead) {
```

Comment:
```suggestion
    if (entriesRead !== undefined) {
```
Would get skipped if `entriesRead` is `0`